images/devcontainer/features.md

```diff
@@ -54,7 +54,7 @@ may depend on which method you wish to use to install the R binary packages.
 
 Some tips for choosing:
 
-- When installing R packages via `apt` or `mamba` (`conda`), dependencies are installed automatically.
+- When installing R packages via `apt` or `mamba` (`conda`), dependencies outside of R are installed automatically.
   But, installing R packages via R function (`install.packages`)
   may require separate `apt` installations of system libraries that are dependencies.
```

Comment:
> But, installing R packages via R function (`install.packages`)  may require separate `apt` installations of system libraries that are dependencies.

You could just add `bspm` to take care ot this.  I do, and I find it has little harm.  (I overdid it in the beginning: it does _not_ need `sudo`.  But `bspm` itself needs to be install via `sudo` and it needs three Python packages.  I recommend it.  Play with `eddelbuettel/r2u:22.04` to see.

Reply:
As mentioned above, there is no need to mention bspm in the documentation because there is no option to configure bspm in the r-apt Feature at current status.
I will come back here when additional option is added to configure bspm.

Reply:
I may not understand Unix very well but when you set up an environment such as a 'dev container',  you control what is installed or not.  If so, why not install `bspm`? 

Many R users know `install.packages()`.   Very few R users know `apt`.

Reply:
I recognize of course that it is a very attractive option.
The issue about this is rocker-org/devcontainer-features#69.

In adding this, I have the following concerns

- The repos option will no longer work, so we will not be able to install packages from repositories such as R-universe in the usual way (correct me if I'm wrong!)
- This is not a problem for Ubuntu using r2u, but for debian, packages that cannot be installed with apt will be source installed.

These two behaviors can be confusing to users and should be noted in the documentation.

> Very few R users know `apt`.

I am aware of this as well, but it seems that there are people who use the r-apt Feature.
https://github.com/rocker-org/devcontainer-features/pkgs/container/devcontainer-features%2Fr-apt

The documentation describes how to install R packages via apt, so users will be able to use it right away.
https://github.com/rocker-org/devcontainer-features/tree/main/src/r-apt#r-package-installation